contribs/gmf/src/directives/mobilegeolocation.js

```diff
@@ -0,0 +1,187 @@
+goog.provide('gmf.MobileGeolocationController');
+goog.provide('gmf.mobileGeolocationDirective');
+
+goog.require('gmf');
+goog.require('ngeo.DecorateGeolocation');
+goog.require('ngeo.FeatureOverlay');
+goog.require('ngeo.FeatureOverlayMgr');
+goog.require('ol.Feature');
+goog.require('ol.Geolocation');
+goog.require('ol.Map');
+goog.require('ol.geom.Point');
+goog.require('ol.proj');
+
+
+/**
+ * Provide a "mobile geolocation" directive.
+ *
+ * @example
+ * <gmf-mobile-geolocation
+ *     gmf-mobile-geolocation-map="ctrl.map"
+ *     gmf-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
+ * </gmf-mobile-geolocation>
+ *
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+gmf.mobileGeolocationDirective = function() {
+  return {
+    restrict: 'E',
+    scope: {
+      'getMobileMapFn': '&gmfMobileGeolocationMap',
+      'getMobileGeolocationOptionsFn': '&gmfMobileGeolocationOptions'
+    },
+    controller: 'GmfMobileGeolocationController',
+    controllerAs: 'ctrl',
+    templateUrl: 'partials/mobilegeolocation.html'
```

Comment:
I personnally don't really care, but we need to be coherent. I just saw that there was inline template in other gmf directives, wondering if there were any reason ?

If you change the makefile for those templates to be managed by `$templateCache`, it's gonna be the same in the end.
So you can keep it like this, maybe we should update other directives like https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/src/directives/search.js#L44